tests/load_from_macro/BUILD

```diff
@@ -24,3 +24,11 @@ py_library(
     # Allow a test to verify an "outside package" doesn't get included
     visibility = ["//examples/wheel:__pkg__"],
 )
+
+genrule(
+    name = "test_current_py_toolchain",
+    srcs = [],
+    outs = ["out.txt"],
+    cmd = "$(PYTHON3) --version > $(location out.txt)",
```

Comment:
Is there a way to use `genrule` and a python toolchain in conjunction with pip deps?

Reply:
cc/ @ f0rmiga @jheaff1

Reply:
See the use of `entry_point` in [pip.md](https://github.com/bazelbuild/rules_python/blob/main/docs/pip.md#pip_parse)

you could use this in a genrule

Reply:
I’m assuming you want to run T he entrypoint of a pip package? If you want to use a pip package as a library, use the `requirement` macro and setup the PYTHONPATH environment variable to include the location of the pip packages. Eg

`PYTHONPATH=$(location {})`.format(requirement(‘the_pip_package`)